subsys/bluetooth/common/log.c

```diff
@@ -55,3 +56,12 @@ const char *bt_addr_le_str_real(const bt_addr_le_t *addr)
 
 	return str;
 }
+
+const char *bt_uuid_str_real(const struct bt_uuid *uuid)
```

Comment:
It is not thread safe. Need at least a comment in documentation.

Reply:
As long as the threads are cooperative threads this is fine. For preemptive threads it is a different story. But so is a lot of things in the host. I don't want to comment this fact all over the place.

Reply:
I still think there should be a word of comment in every function that is not thread safe - you cannot stop user from using in interrupts nor in pre-emptive threads without it. But I might agree that if there is more such functions - it is a topic for another issue.

Reply:
Added comment to the header file.